Packs/ServiceNow/ReleaseNotes/2_2_4.md

```diff
@@ -0,0 +1,8 @@
+#### Integrations
+
+##### ServiceNow v2
+
+- When creating notes in XSOAR, the integration displays the username of the user that created the note, however when an
+  automation creates a note, it shows up as dbot. Now, this will also happen for instances where the user field is populated with
```

Comment:
```suggestion
  automation creates a note, it shows up as "dbot". Now, this will also happen for instances where the user field is populated with
```